stream-chat-android-client-test/api/stream-chat-android-client-test.api

```diff
@@ -65,8 +65,8 @@ public final class io/getstream/chat/android/client/test/MotherKt {
 	public static synthetic fun randomTypingStartEvent$default (Ljava/lang/String;Ljava/util/Date;Lio/getstream/chat/android/client/models/User;Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;ILjava/lang/Object;)Lio/getstream/chat/android/client/events/TypingStartEvent;
```

Comment:
Is there a point to generating an api file for tests or is this something we forgot to exclude?

Reply:
I believe it is something we forgot to exclude.